OptimizelySDK/OptlyConfig/OptimizelyConfigService.cs

```diff
@@ -31,16 +33,39 @@ public OptimizelyConfigService(ProjectConfig projectConfig)
             {
                 return;
             }
+            var events = GetEvents(projectConfig);
+            var attributes = GetAttributes(projectConfig);
+            var audiences = GetAudiences(projectConfig).Select(aud => new OptimizelyAudience(aud.Id, aud.Name, aud.Conditions)).ToArray<OptimizelyAudience>();
             var experimentMap = GetExperimentsMap(projectConfig);
             var featureMap = GetFeaturesMap(projectConfig, experimentMap);
             OptimizelyConfig = new OptimizelyConfig(projectConfig.Revision,
                 projectConfig.SDKKey,
                 projectConfig.EnvironmentKey,
+                attributes,
+                audiences,
+                events,
                 experimentMap,
                 featureMap,
                 projectConfig.ToDatafile());
         }
 
+        private Entity.Event[] GetEvents(ProjectConfig projectConfig)
+        {
+            return projectConfig.Events;
+        }
+
+        private Audience[] GetAudiences(ProjectConfig projectConfig)
+        {
+            var audiencesArr = Array.FindAll(projectConfig.Audiences, aud => !aud.Id.Equals("$opt_dummy_audience"));
```

Comment:
am just thinking, can't we avoid this condition? may be we should remove somewhere else.